.ci/lib.sh

```diff
@@ -100,20 +123,44 @@ function install_yq() {
 	fi
 }
 
+function get_dep_from_yaml_db(){
+	local versions_file="$1"
+	local dependency="$2"
+
+	[ ! -f "$versions_file" ] && die "cannot find $versions_file"
+
+	# This is needed in order to retrieve the version for qemu-lite
```

Comment:
I think this comment can now be removed...

Reply:
Zapped! :smile: